main/wallet-api.md

```diff
@@ -0,0 +1,206 @@
+
+### `getPurseIssuer(pursePetname)`
+- **Function:**
+  - Given a purse's petname, returns the `issuer` object associated
+    with the purse.
+- **Required Arguments:**
+  - **String**. `petname`, a purse's petname
+- **Optional Arguments:**
+  - None.
+- **Errors**
+  - Throws and error if given an invalid purse petname. 
+
+### `makeEmptyPurse(issuerPetname, pursePetname)`
+- **Function:**
+  - Creates and returns a new, empty, purse object with a petname specified by the command. 
+- **Required Arguments:**
+  - `issuerPetname`
+    - **String**. The petname you've assigned to the `issuer` the new purse uses.
+  - `pursePetname`
+    - **String**. The petname assigned to the new purse.	
+- **Optional Arguments:**
+ - None.
+- **Errors:**
+  - There is already a purse in this wallet with the name of `pursePetname`'s value.   
+
+
+## Issuer API Methods
+
+### `addIssuer()`
+- **Function:**
+  -  Assigns the given petname to a given`issuer` object and adds the issuer to the wallet. 
+- **Required Arguments:**
+  - `issuerPetname`
+    - **String**. The petname assigned to the `issuer` object.
+  - `issuer`
+    - **Object**. An `issuer` object.
+- **Optional Arguments:**
+  - `brandRegKey`
+    - **String**. A key in the Registry, whose value is the `brand` object associated with the issuer.
+
+### `getIssuers()`
+- **Function:**
+  - Returns all issuers associated with the wallet and their petnames as an array of arrays of the format `[[issuerPetname, issuer], ...]`. You can use it to make a new map of petnames to issuers by doing `new Map(getIssuer())` 
+- **Required Arguments:**
+  - None.
+- **Optional Arguments:**
+  - None.
+
+### `getIssuerNames(issuer)`
+- **Function:**
+  - Returns the petname and brandRegKey of a particular issuer object
+- **Required Arguments:**
+  - `issuer`
+    - An issuer object
+- **Optional Arguments:**
+  - None.
+
+## Offer API Methods
+
+### `acceptOffer()`
+- **Function:**
+  - Changes the specified offer's status to "accepted".
+- **Required Arguments:**
+  - `id`
+    - **String**: An offer id.
+- **Optional Arguments:**
+  - None.
+- **Errors:**
+  - If the offer has already been resolved or rejected. 
+
+### `addOffer()`  
+- **Function:**
+  - Adds a new offer to the wallet's inbox, returning the offer's id.
+- **Required Arguments:**
+  - `rawOffer` 
+    - **Object**: The potential offer sent from the Dapp UI to be looked at by the user.
+- **Optional Arguments:**
+  - `hooks`
+    - Instructs the wallet to do actions before or after the offer is made, such as getting an invite from the `publicAPI` in order to make an offer. 
+
+`hooks` format is 
+
+`hooks[targetName][hookName] = [hookMethod, ...hookArgs]`
+
+It’s called within the wallet as
+`E(target)[hookMethod](...hookArgs)`
+
+`hooks: {
+     publicAPI: {
+        getInvite: ['makeInvite'], // E(publicAPI).makeInvite()
+     },
+}`
+  
+### `cancelOffer()`
+- **Function:**
+  - Changes the status of the specified offer in the wallet inbox to
```

Comment:
The "changes the status" is not the important thing. It's "transition the state" by e.g., submitting or canceling the offer in the contract.